Language link placement (#62)
ropensci-review-tools/babelquarto: README.Rmd

Using `babelquarto::register_main_language()` and `babelquarto::register_further_languages()` will create the boilertemplate for these fields.

## Configure the language link placement

> The language link menu can be placed in the *sidebar* or in the *navbar*. To change the placement, use the languagelinks field in the babelquarto YAML key, in `_quarto.yml`:

Reviewer: ```suggestion
The language link menu can be placed in the *sidebar* or in the *navbar*. 
To change the placement, use the languagelinks field in the babelquarto YAML key, in `_quarto.yml`:
```

In Markdown, adding linebreaks at the end of sentences does not change the resulting HTML but it makes reviews and diffs easier.

```yaml
babelquarto:
  languagelinks: sidebar
```

> Beware: depending on the languagelinks' value, the corresponding menu needs to be configured. Books always have a *sidebar*, but websites can have either *navbar* or *sidebar*.

Reviewer: ```suggestion
Beware: depending on the value of `languagelinks`, the corresponding menu needs to be configured. 
Books always have a *sidebar*, but websites can have either *navbar* or *sidebar*.
```

Add link to https://quarto.org/docs/websites/website-navigation.html#side-navigation?

Author: I added the link to quarto's documentation.

Using `babelquarto::register_main_language()` and `babelquarto::register_further_languages()` will create the boilertemplate for these fields.

## Configure the language link placement

The language link menu can be placed in the *sidebar* or in the *navbar*. 

> To change the placement, use the languagelinks field in the babelquarto YAML key, in `_quarto.yml`:

Reviewer: ```suggestion
To change the placement, use the `languagelinks` field in the babelquarto YAML key, in `_quarto.yml`:
```
